src/bentoml/_internal/cloud/deployment.py

```diff
@@ -52,6 +119,80 @@ def create_deployment(
             raise BentoMLException("Create deployment request failed")
         return res
 
+    def create_deployment(
+        self,
+        deployment_name: str,
+        bento_repository: Tag,
+        runner_config: dict[str, DeploymentConfig],
+        enable_ingress: bool,
+        mode="function",
+        type=DeploymentTargetType.STABLE,
+        context: str | None = None,
+        clusterName: str | None = None,
+        kube_namespace: str | None = None,
+        api_server_config: DeploymentConfig | None = None,
+        bento_request_overrides: dict[str, t.Any] | None = None,
+        do_not_deploy: bool = False,
+        description: str = "",
```

Comment:
Make this None and set to empty string inside implementation. Doesn't make sense to set it here.